### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by hellais**
Some improvements to measure performance of the API better and possibly boost it.

### CODE REVIEW

File: measurements/gunicorn_config.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from .config import PROMETHEUS_PORT
```
Comment (bassosimone) [inline measurements/gunicorn_config.py]: I'd probably mention that this is the vanilla usage according to the docs

File: measurements/wsgi.py [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
 from werkzeug.contrib.fixers import ProxyFix
 
 from measurements.app import create_app
 
+from prometheus_flask_exporter.multiprocess import GunicornPrometheusMetrics
+
 application = create_app()
 application.wsgi_app = ProxyFix(application.wsgi_app)
+
+metrics = GunicornPrometheusMetrics(application, group_by='endpoint')
```
Comment (bassosimone) [inline measurements/wsgi.py]: I assume you will use `metrics` later?

Comment (hellais) [reply measurements/wsgi.py]: There is stuff that can be done by using metrics, but I don't need to currently. I put it inside of a variable so we don't forget that this instance can be used to further customise the metrics endpoint.

File: measurements/gunicorn_config.py (hunk 1 above)
Comment (hellais) [reply measurements/gunicorn_config.py]: What do you mean by this?

Comment (bassosimone) [reply measurements/gunicorn_config.py]: That this code is exactly the default way of deploying if you read the docs.

**Pull request merged**